app/services/hyrax/indexes_thumbnails.rb

```diff
@@ -0,0 +1,37 @@
+# OVERRIDE Hyrax 2.9.0 to make collection thumbnails uploadable
```

Comment:
in the comments below, can you explicitly state "OVERRIDE: ...." for the overridden code that correlates to this comment?